tests/flow/tests_dag.py

```diff
@@ -430,48 +288,33 @@ def test_dagro_local_tensorset(env):
     ret = con.execute_command(command)
     env.assertEqual(ret, [b'OK',b'OK'])
 
-    # assert that transaction tensor does not exist
+    # assert that volatile_tensor does not exist
     ret = con.execute_command("EXISTS volatile_tensor")
     env.assertEqual(ret, 0 )
 
 
 def test_dag_local_tensorset_persist(env):
     con = env.getConnection()
 
-    command = "AI.DAGRUN "\
+    command = "AI.DAGEXECUTE "\
         "PERSIST 1 tensor1{{1}} |> "\
         "AI.TENSORSET tensor1{{1}} FLOAT 1 2 VALUES 5 10"
 
     ret = con.execute_command(command)
     env.assertEqual(ret, [b'OK'])
 
-    # assert that transaction tensor exists
+    # assert that PERSIST succeeded.
     ret = con.execute_command("EXISTS tensor1{{1}}")
     env.assertEqual(ret, 1 )
 
     ret = con.execute_command("AI.TENSORGET tensor1{{1}} META VALUES")
     env.assertEqual(ret, [b'dtype', b'FLOAT', b'shape', [1, 2], b'values', [b'5', b'10']])
 
 
-def test_dagro_local_tensorset_persist(env):
-    con = env.getConnection()
-
-    command = "AI.DAGRUN_RO "\
-        "PERSIST 1 tensor1{{1}} |> "\
-        "AI.TENSORSET tensor1{{1}} FLOAT 1 2 VALUES 5 10"
-
-    try:
-        con.execute_command(command)
-    except Exception as e:
-        exception = e
-        env.assertEqual(type(exception), redis.exceptions.ResponseError)
-        env.assertEqual("PERSIST cannot be specified in a read-only DAG", exception.__str__())
```

Comment:
why was this deleted?

Reply:
This has check has moved to another test called `test_dag_ro_errors`